root/help.1

```diff
@@ -0,0 +1,83 @@
+.TH "DHCP-CLIENT " "1" " Container Image Pages" "Karsten Hopp" "June 13, 2017"  ""
+
+
+.SH NAME
+.PP
+dhcp\-client \- Dynamic Host Configuration Protocol Client
+
+
+.SH DESCRIPTION
+.PP
+A DHCP client container running as a root user based on Base Runtime. Provides a means for configuring one or more network interfaces using DHCP.
+
+
+.SH USAGE
+.PP
+This container can be run as a service by running:
+
+.PP
+.RS
+
+.nf
+# systemctl start dhcp\-client\-container
```

Comment:
I would remove this line since we didn't automate obtaining systemd unit files. (feel free to mention it in readme though, since the unit file is present in the repository)